core/modules/util/Error.cc

```diff
@@ -0,0 +1,56 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2014 LSST Corporation.
```

Comment:
Would claiming copyright for 2008 in files that were created 6 years later be appropriate?